src/content/vintage/use-the-api/kubectl-gs/update-cluster.md

```diff
@@ -52,9 +52,34 @@ kubectl gs update cluster \
 
 This adds annotations to the cluster resource, triggering the upgrade to the specified version at the scheduled time.
 
+### Upgrading a workload cluster {#cluster-upgrade} for CAPI cluster
+
+**Note:** This feature is currently only working when the cluster configuration is **not stored** via GitOps.
+
+To upgrade a workload cluster immediately, use the following command:
+
+```nohighlight
+kubectl gs update cluster \
+  --provider capa \
+  --namespace org-acme \
+  --name a1b2c \
+  --release-version 16.1.0
+```
+
+To schedule a workload cluster upgrade in the future, the `--scheduled-time` flag is used, like in the example below:
```

Comment:
Is this already supported?

Reply:
**Note:** This feature is currently only working when the cluster configuration is **not stored** via GitOps.

I mentioned it above updating cluster is only working when not using GitOps